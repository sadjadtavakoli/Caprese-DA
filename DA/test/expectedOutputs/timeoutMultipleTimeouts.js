
#2 function setTimeout21 is called with variables args by timeoutMultipleTimeouts.js
#4 function setTimeout41 is called with variables args by timeoutMultipleTimeouts.js
#6 function setTimeout61 is called with variables args by timeoutMultipleTimeouts.js
#8 function setTimeout81 is called with variables args by timeoutMultipleTimeouts.js
#10 function myFunction entered with variables args from setTimeOut6
#10 function myFunction exited with return values undefined to function setTimeOut6
#10 function myFunction entered with variables args from setTimeOut8
#10 function myFunction exited with return values undefined to function setTimeOut8
#10 function myFunction entered with variables args from setTimeOut4
#10 function myFunction exited with return values undefined to function setTimeOut4
#10 function myFunction entered with variables args from setTimeOut2
#10 function myFunction exited with return values undefined to function setTimeOut2
#end Execution