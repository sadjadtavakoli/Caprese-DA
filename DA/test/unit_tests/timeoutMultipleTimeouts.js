
setTimeout(myFunction, 1000);

setTimeout(myFunction, 500);

setTimeout(myFunction, 100);

setTimeout(myFunction, 100);

function myFunction() {
//  do nothing
}

