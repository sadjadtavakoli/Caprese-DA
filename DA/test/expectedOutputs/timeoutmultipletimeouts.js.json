{"timeoutmultipletimeouts.js-test/unit_tests/timeoutmultipletimeouts.js-1-13":{"impacted":["t_3"]},"keyMap":{"t_1":"timeoutmultipletimeouts.js-test/unit_tests/timeoutmultipletimeouts.js-1-13","t_3":"myFunction-test/unit_tests/timeoutmultipletimeouts.js-10-12"}}